### PR TITLE
Feature/transparency

### PR DESCRIPTION
**Pull request by netbe**
Ported changes mentioned in #266 for Koloda 4.0 swift 3.0

Please let me know if this can be merged or need some changes.

### CODE REVIEW

**Comment by blender** [comment]
Hi,

are there any plans on merging this pull request? Changes allows for greater flexibility and everyone could benefit from it.

**Comment by netbe** [comment]
Hi @freaqed, would it be possible to have a reply from Yalantis' team on this?

**Comment by Tykhonkov** [comment]
Hi, @netbe I have reviewed and fixed conflicts of your PR, generally it's good, but there is several issues with code formatting and issue with blinking effect when user cancels card swipe. I will comment style issues but the blinking effect is up to you.
Also we should not change default behavior of Koloda, so it would be great if you will fix default transparency from this 
Preview
 to this 
Preview

File: Pod/Classes/KolodaView/KolodaView.swift [hunk 1]
```diff
@@ -344,7 +346,14 @@ open class KolodaView: UIView, DraggableCardDelegate {
         visibleCards.removeAll(keepingCapacity: true)
     }
     
-    // MARK: Actions
+    private func cardAlphaTransparent(at index: Int) -> CGFloat {
+        
+        let alphaTransparent: CGFloat = self.delegate != nil ? self.delegate!.kolodaOpacityForNextCard(self, at: index) : alphaValueSemiTransparent
```
Comment (Tykhonkov) [inline Pod/Classes/KolodaView/KolodaView.swift]: you can simplify it
```swift
let alphaTransparent = self.delegate?.kolodaOpacityForNextCard(self, at: index) ?? alphaValueSemiTransparent
```

File: Pod/Classes/KolodaView/KolodaViewAnimatior.swift [hunk 2]
```diff
@@ -107,6 +108,11 @@ open class KolodaViewAnimator {
             }
         )
     }
+
+	open func removeCardAnimation(card: DraggableCardView, completion: AnimationCompletionBlock = nil) {
```
Comment (Tykhonkov) [inline Pod/Classes/KolodaView/KolodaViewAnimatior.swift]: wrong formatting

**Comment by rnkyr** [comment]
hello guys!
looks like the MR is no longer needed. if you still need these changes to be merged please resolve conflicts and comments and re-open it
thanks!

**Pull request closed without merging**